stream-chat-android-ui-components/src/main/res/values/strings.xml

```diff
@@ -141,11 +141,12 @@
 
     <string name="stream_ui_reply_to_message">Reply to message</string>
 
-    <!-- AttachmentGallery -->
-    <string name="stream_ui_images_menu_default_title">Photos</string>
-
-    <!-- Gallery -->
+    <!-- Attachment Gallery -->
+    <string name="stream_ui_gallery_overview_title">Photos</string>
     <string name="stream_ui_gallery_share_sheet_title">Share Image</string>
+    <string name="stream_ui_gallery_count_template"> %d - %d</string>
```

Comment:
`%d of %d` is the desired default. I substituted just to show that configuration was possible.

Reply:
Nice catch, will replace with %d of %d. Also noticed that 1 of 1 is also shown in the design. I guess we were hiding the label in that case. Fix fix that as well.